client/signup/steps/emails/index.jsx

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'calypso/signup/step-wrapper';
+import EmailTitanCard from 'calypso/components/emails/emails-titan-card';
+import EmailSideExplainer from 'calypso/components/emails/emails-side-explainer';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import CalypsoShoppingCartProvider from 'calypso/my-sites/checkout/calypso-shopping-cart-provider';
+
+class EmailsStep extends React.Component {
+	constructor( props ) {
+		super( props );
+	}
+
+	getSideContent = () => {
+		return (
+			<div className="emails__email-side-content-container">
+				<div className="emails__email-side-content">
+					<EmailSideExplainer />
+				</div>
+			</div>
+		);
+	};
+
+	renderContent() {
+		const content = (
+			<div className="register-email-step">
+				<CalypsoShoppingCartProvider>
+					<EmailTitanCard
+						siteUrl={ this.props.progress.domains.siteUrl }
+						addButtonContent={ 'Add' }
+						skipButtonContent={ 'Skip' }
+						onAddButtonClick={ () => {} }
+						onSkipButtonClick={ () => {} }
+						price={ '3.5' }
+					></EmailTitanCard>
+				</CalypsoShoppingCartProvider>
+			</div>
+		);
+		const sideContent = this.getSideContent();
+
+		return (
+			<div
+				key={ this.props.step + this.props.stepSectionName }
+				className="emails__step-content emails__step-content-email-step"
+			>
+				{ content }
+				{ sideContent }
+			</div>
+		);
+	}
+
+	handleSkip = () => {};
+
+	render() {
```

Comment:
Stylistic note: I find the jumping across the three rendering functions harder to follow, especially because there is very little logic in each function. They each set up some variables and then return JSX. As I noted elsewhere, I prefer those pieces to be sequential in the code, even if it does increase the function size a bit, because I can then see what pieces are going in to the returned JSX fairly easily. (This is not an instruction to change, BTW, more a comment on how I am noticing my preference! We should discuss before you make changes! )